docs/colorpicker.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Hyper-Link DescribedBy JSON Editor Example</title>
```

Comment:
Looks good. just a few minor changes left,,
The example header text is wrong.

Reply:
> I noticed an odd thing.
> When I open the example `docs/colorpicker.html`, the ccolorpicker of the 1st item is open without me clicking on the form field,
> ![image](https://user-images.githubusercontent.com/478711/67619496-cdea1480-f7fc-11e9-83ec-ee7994bc3aac.png)
> Might be because I run it from `file:///C:/` and not `http://` .I will test it in a server setup later.

It was cause by I Call the `OpenHandler` member of `Picker` instance when `Picker` instance created, which is used for openning the ColorPicker, it have been removed now.